doc/guides/maintaining-V8.md

```diff
@@ -289,17 +294,13 @@ To audit for floating patches:
 git log --oneline deps/v8
 ```
 
-To replace the copy of V8 in Node.js, use the `[update-v8](https://gist.github.com/targos/8da405e96e98fdff01a395bed365b816)` script<sup>2</sup>. For example, if you want to replace the copy of V8 in Node.js with the branch-head for V8 5.1 branch:
+To replace the copy of V8 in Node.js, use the [`update-v8`](https://github.com/targos/update-v8) tool. For example, if you want to replace the copy of V8 in Node.js with the branch-head for V8 5.1 branch:
```

Comment:
Split to 80 char line lengths while you're here?

Reply:
done. while I was there, I split the entire document to 80 char line lengths and put the links at the bottom.